Documentation/network/concepts/masquerading.rst

```diff
@@ -31,6 +31,17 @@ Setting the routable CIDR
   10.0.0.0/8`` (or ``ipv6-native-routing-cidr: fd00::/100`` for IPv6 addresses)
   in which case all destinations within that CIDR will **not** be masqueraded.
 
+.. note::
+
```

Comment:
Please drop the `.. note::`. This entire document is already about the configuration and behavior of the masquerade feature, so it's better to just describe the behavior in a consistent format. See the [docs style](https://docs.cilium.io/en/stable/contributing/docs/docsstyle/#callouts) for more details.

```suggestion
Masquerade behavior in cloud environments

```

Reply:
It was my first version where I didn't put it in notes and I found configuration section talking about something you can configure Setting the routable CIDR or Setting the masquerading interface. my description is more like the note than something you can configure